.idea/runConfigurations/test.xml

```diff
@@ -1,27 +1,18 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="test" type="tests" factoryName="py.test">
+    <module name="Lexos" />
     <option name="INTERPRETER_OPTIONS" value="" />
     <option name="PARENT_ENVS" value="true" />
-    <envs />
-    <option name="SDK_HOME" value="$USER_HOME$/anaconda3/bin/python" />
+    <option name="SDK_HOME" value="$PROJECT_DIR$/../anaconda3/bin/python" />
     <option name="WORKING_DIRECTORY" value="$PROJECT_DIR$" />
     <option name="IS_MODULE_SDK" value="false" />
     <option name="ADD_CONTENT_ROOTS" value="true" />
     <option name="ADD_SOURCE_ROOTS" value="true" />
-    <module name="Lexos" />
-    <EXTENSION ID="PythonCoverageRunConfigurationExtension" enabled="false" sample_coverage="true" runner="coverage.py" />
-    <option name="SCRIPT_NAME" value="" />
-    <option name="CLASS_NAME" value="" />
-    <option name="METHOD_NAME" value="" />
-    <option name="FOLDER_NAME" value="" />
-    <option name="TEST_TYPE" value="TEST_SCRIPT" />
-    <option name="PATTERN" value="" />
-    <option name="USE_PATTERN" value="false" />
-    <option name="testToRun" value="" />
-    <option name="keywords" value="" />
-    <option name="params" value="" />
-    <option name="USE_PARAM" value="false" />
-    <option name="USE_KEYWORD" value="false" />
+    <EXTENSION ID="PythonCoverageRunConfigurationExtension" runner="coverage.py" />
+    <option name="_new_keywords" value="&quot;&quot;" />
+    <option name="_new_additionalArguments" value="&quot;&quot;" />
```

Comment:
I don't quite understand the following 3 lines. Please review and make sure they are okay.

Reply:
I think they are fine. There were keywords and additional arguments settings in the pytest and I left them empty.